### PR TITLE
Removequiz

### PR DESCRIPTION
**Pull request by coati-00**
I removed functionality for storing user information. Anonymous users can go through the material.

### CODE REVIEW

File: phtc/logic_model/views.py [hunk 1]
```diff
@@ -1,14 +1,10 @@
-from django.contrib.auth.decorators import login_required
-from django.http import HttpResponse, HttpResponseForbidden
+import simplejson
```
Comment (thraxil) [inline phtc/logic_model/views.py]: why pull in simplejson?

File: phtc/main/views.py [hunk 2]
```diff
@@ -695,15 +157,15 @@ def render_dashboard(request):
 
     h = get_hierarchy("main")
     root = h.get_root()
-    last_session = h.get_user_section(request.user)
+    # last_session = h.get_user_section(request.user)
```
Comment (thraxil) [inline phtc/main/views.py]: remove commented code

File: phtc/logic_model/views.py (hunk 1 above)
Comment (coati-00) [inline phtc/logic_model/views.py]: It seems to be using it to create the response.

Comment (sdreher) [inline phtc/logic_model/views.py]: Django 1.5 deprecates django.utils.simplejson in favor of Python 2.6’s built-in json module. You can just swap out "simplejson" for "json".
https://docs.djangoproject.com/en/1.8/releases/1.5/#system-version-of-simplejson-no-longer-used

Comment (coati-00) [inline phtc/logic_model/views.py]: okay great

**Pull request merged**